_includes/sitemenu.html

```diff
@@ -26,9 +26,12 @@ <h3 class="sitemenu__apititle epsilon u-mb0"><a href="{{ site.baseurl }}/api/ser
             </ul>
             {% endif %}
           </li>
-          <li class="sitemenu__separator {% if page.url == '/revision-history/' %} active{% endif %}">
+          <li class="{% if page.url == '/revision-history/' %} active{% endif %}">
             <h3 class="sitemenu__apititle epsilon u-mb0"><a href="{{ site.baseurl }}/api/revision-history/">Revision History</a></h3>
           </li>
+          <li class="sitemenu__separator">
+            <h3 class="sitemenu__apititle epsilon u-mb0"><a href="https://status.bring.systems" target="_blank">API Status</a></h3>
```

Comment:
```suggestion
            <h3 class="sitemenu__apititle epsilon u-mb0"><a href="https://status.bring.systems" target="_blank" rel="noreferrer">API Status</a></h3>
```
Since `target="_blank"` is considered a security issue: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
Again...I forgot this! This is the second time. 

Thank you. I'll fix.